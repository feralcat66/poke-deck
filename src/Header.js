import React, { Component } from 'react';

export default class Header extends Component {

    render() {
        return(
          <h1><img src="https://fontmeme.com/permalink/200213/cda48847e72671ec952d89973b268209.png" alt="pokemon-font" border="0"/></h1>  
    
        )
    }
}