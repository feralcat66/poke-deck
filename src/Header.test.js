import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header.js'

test ('this renders header', () => {
    const header = renderer
    .create(<Header />)
    .toJSON();
    expect (header).toMatchSnapshot();
})