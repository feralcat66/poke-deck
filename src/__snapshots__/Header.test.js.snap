// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this renders header 1`] = `
<h1>
  <img
    alt="pokemon-font"
    border="0"
    src="https://fontmeme.com/permalink/200213/cda48847e72671ec952d89973b268209.png"
  />
</h1>
`;
